solr/core/src/test/org/apache/solr/handler/JsonLoaderTest.java

```diff
@@ -435,61 +435,80 @@ public void testJsonDocFormat() throws Exception{
 
   public void testFewParentsJsonDoc() throws Exception {
     String json = PARENT_TWO_CHILDREN_JSON;
+    assertFewParents(json, true);
+    tearDown();
```

Comment:
I'm don't think I've ever seen a test do this, and I don't think we should start now.  Simply add two test methods, one that passes true, one that passes false.  test methods need to be named differently of course.  This pattern in testing is pretty common.